### DISCUSSION TITLE
How to change font size in aterm?

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-packages#17510] [Q&A]
Moved here as instructed. How to change the font size in aterm because the default ones is very small?

**Comment by RalfWerner** [comment]
> Moved [3525 ](https://github.com/termux/termux-app/discussions/3524#discussion-5438424) here as instructed.

@FlcristPony `aterm` is a XClient and need a XServer (you used **Xvnc**). In your Shot is additionally a **WM**=Window Manager (**openbox**) used and you need a VNC reader (possibly RealVNC). to get Infos to parameters (size/font/geometry) use `aterm -help`

If your question is: what is the most effective terminal-view can be used on the small display of a phone, there may be completely different answers. **Termux** is for me the most effektive one

**Comment by sylirre** [comment]
The font size as well as its other properties can be specified by `-fn` option.
